farm/evaluation/metrics.py

```diff
@@ -156,7 +158,22 @@ def squad_EM(preds, labels):
         curr_labels = labels[doc_idx]
         if (pred_start, pred_end) in curr_labels:
             n_correct += 1
-    return n_correct/n_docs
+    return n_correct/n_docs if n_docs else 0
+
+def squad_EM_start(preds, labels):
+    """
+    Count how often the predicted start index exactly matches the start index given by one of the labels
+    """
+    n_docs = len(preds)
+    n_correct = 0
+    for doc_idx in range(n_docs):
```

Comment:
just suggestion, we could use `zip` as well -
```
for curr_labels, pred in zip(labels , preds):
```

Reply:
Good suggestion. It's more consistent to use `zip` here as well and I don't see any reason for not using it. 👍